frontend/ui-pablo/storybook/stories/organisms/auction_histories_table.stories.tsx

```diff
@@ -7,7 +7,7 @@ const AuctionHistoriesTableStories = () => {
   const {auctions} = useStore();
   return (
     <Box>
-      <AuctionHistoriesTable auction={auctions.activeLBP} />
+      <AuctionHistoriesTable auction={auctions.activeLBP} baseAsset={undefined} quoteAsset={undefined} />
```

Comment:
explicit `undefined` can be omitted here by changing `baseAsset` prop to `baseAsset?: .....` and similarly in `quoteAsset`

Reply:
updated